aws/ecs/cluster/vars.tf

```diff
@@ -24,3 +24,9 @@ variable "amiFilter" {
   type    = string
   default = "amzn2-ami-ecs-hvm-*-x86_64-ebs"
 }
+
+variable "tags" {
+  type        = map(any)
+  description = "Map of tags to add to the rds instance. Duplicate tags will be overridden."
+  default     = {}
+}
```

Comment:
This file is missing the ending newline character. (That's what the "do not enter" symbol means.)